core/modules/catalog/store/tax/getters.ts

```diff
@@ -0,0 +1,18 @@
+import { GetterTree } from 'vuex'
+import RootState from '@vue-storefront/core/types/RootState'
+import TaxState from '../../types/TaxState'
+
+const getters: GetterTree<TaxState, RootState> = {
+  getUserTaxGroupId: (state, getters, rootState) => {
```

Comment:
not too sure if that supposed to be a part of user module, i thing both places are ok (user or tax)

Reply:
it should be more in tax